mesonbuild/modules/gnome.py

```diff
@@ -45,7 +45,11 @@ def compile_resources(self, state, args, kwargs):
         if not isinstance(source_dirs, list):
             source_dirs = [source_dirs]
 
-        ifile = args[1]
+        try:
+            ifile = args[1]
+        except IndexError:
```

Comment:
It would be clearer to check the length of `args` instead of doing `try`...`except`.